src/api.ts

```diff
@@ -125,14 +127,15 @@ export const apiFactory = (serviceId: number) => {
 				'openvpn-oos-event',
 				{ tags: { uuid }, req },
 			);
-			return res.sendStatus(400);
+			return res.status(400).end();
 		}
+		res.status(200).end();
```

Comment:
Moving this before the `metrics.dec`/`clients.setConnected` is the biggest win here